docs/tutorials/vagrant-setup.rst

```diff
@@ -0,0 +1,202 @@
+RackHD: local Vagrant based setup
+==================================
+
+This tutorial gets an instance of RackHD up and running on your local desktop or
+laptop, so you can see the hosted API documentation and experiment with the APIs.
+
+prerequisites
+--------------
+
+.. sidebar:: jq
+
+    You can get details on how to use `jq`_ at https://stedolan.github.io/jq/manual/.
+    By default it colorizes the syntax highlighting and pretty prints javascript data structures.
+
+    Another option that can provide the same pretty printing is use::
+
+        | python -mjson.tool
+
+You will need to have `Vagrant`_ (>=1.8.1) and `VirtualBox`_ (4.3.x) installed on your machine to use
+this tutorial. The 5.1.X `VirtualBox`_ has some problems. So, you are not suggested installing 5.1.x `VirtualBox`_.
```

Comment:
The 5.1.X `VirtualBox`_ has some problems. So, you are not suggested installing 5.1.x `VirtualBox ?